tests/reader-config.rs

```diff
@@ -0,0 +1,44 @@
+//! Contains tests for config options of a parser.
+//!
+//! Each module has a name of a corresponding option and functions inside performs
+//! testing of various option values.
+//!
+//! Please keep tests sorted (exceptions are allowed if options are tightly related).
+
+use quick_xml::events::{BytesEnd, BytesStart, Event};
+use quick_xml::reader::Reader;
+
+mod expand_empty_elements {
+    use super::*;
+    use pretty_assertions::assert_eq;
+
+    /// Self-closed elements should be reported as one `Empty` event
+    #[test]
+    fn false_() {
```

Comment:
Perhaps "enabled" and "disabled" would be better names, to avoid keyword clashes?

Reply:
I chose those words because if we will have other options that are not booleans, tests would be named as a config value